MultiPoolMiner.ps1

```diff
@@ -59,7 +59,7 @@ param(
     [Alias("Uri", "Url")]
     [String]$MinerStatusUrl = "", #i.e https://multipoolminer.io/monitor/miner.php
     [Parameter(Mandatory = $false)]
-    [String]$MinerStatusKey = "",
+    [String]$MinerStatusKey = $Wallet, #For backwards compatibility, set the MinerStatusKey to $Wallet if it's not specified
```

Comment:
I don't think you can do this...

Reply:
Yep - it works :-)
This is how I tested:
- Remove config file
- Run default Start.bat
- Launch API web page

This is what I get:
```
{
    "VersionCompatibility":  "3",
    "Wallet":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
    "UserName":  "aaronsace",
    "WorkerName":  "multipoolminer",
    "API_ID":  0,
    "API_Key":  "",
    "Interval":  60,
    "ExtendIntervalAlgorithm":  [
                                    "X16R",
                                    "X16S"
                                ],
    "ExtendIntervalMinerName":  "PalginNvidia",
    "Region":  "Europe",
    "SSL":  {
                "IsPresent":  false
            },
    "DeviceName":  [
                       "amd",
                       "nvidia",
                       "cpu"
                   ],
    "Algorithm":  [
                      "Blake2s",
                      "CryptoNightV7",
                      "Decrednicehash",
                      "Ethash",
                      "Ethash2gb",
                      "Equihash",
                      "Groestl",
                      "Keccak",
                      "Lbry",
                      "Lyra2RE2",
                      "Lyra2z",
                      "NeoScrypt",
                      "Pascal",
                      "Sib",
                      "Skunk"
                  ],
    "MinerName":  null,
    "PoolName":  [
                     "miningpoolhubcoins",
                     "zpool",
                     "nicehash"
                 ],
    "ExcludeAlgorithm":  null,
    "ExcludeMinerName":  null,
    "ExcludePoolName":  null,
    "Currency":  [
                     "BTC",
                     "USD",
                     "EUR"
                 ],
    "Donate":  24,
    "Proxy":  "",
    "Delay":  0,
    "Watchdog":  {
                     "IsPresent":  true
                 },
    "MinerStatusUrl":  "https://multipoolminer.io/monitor/miner.php",
    "MinerStatusKey":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
    "SwitchingPrevention":  2,
    "DisableAutoUpdate":  {
                              "IsPresent":  false
                          },
    "ShowMinerWindow":  {
                            "IsPresent":  false
                        },
    "UseFastestMinerPerAlgoOnly":  {
                                       "IsPresent":  false
                                   },
    "IgnoreCosts":  {
                        "IsPresent":  false
                    },
    "ShowPoolBalances":  {
                             "IsPresent":  false
                         },
    "ShowPoolBalancesForExcludedPools":  {
                                             "IsPresent":  true
                                         },
    "Pools":  {
                  "AHashPool":  {
                                    "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                    "User":  "aaronsace",
                                    "Worker":  "multipoolminer",
                                    "API_ID":  0,
                                    "API_Key":  ""
                                },
                  "BlazePool":  {
                                    "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                    "User":  "aaronsace",
                                    "Worker":  "multipoolminer",
                                    "API_ID":  0,
                                    "API_Key":  ""
                                },
                  "BlockMasters":  {
                                       "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                       "User":  "aaronsace",
                                       "Worker":  "multipoolminer",
                                       "API_ID":  0,
                                       "API_Key":  ""
                                   },
                  "Hashrefinery":  {
                                       "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                       "User":  "aaronsace",
                                       "Worker":  "multipoolminer",
                                       "API_ID":  0,
                                       "API_Key":  ""
                                   },
                  "MiningPoolHub":  {
                                        "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                        "User":  "aaronsace",
                                        "Worker":  "multipoolminer",
                                        "API_ID":  0,
                                        "API_Key":  ""
                                    },
                  "MiningPoolHubCoins":  {
                                             "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                             "User":  "aaronsace",
                                             "Worker":  "multipoolminer",
                                             "API_ID":  0,
                                             "API_Key":  ""
                                         },
                  "NiceHash":  {
                                   "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                   "User":  "aaronsace",
                                   "Worker":  "multipoolminer",
                                   "API_ID":  0,
                                   "API_Key":  ""
                               },
                  "PhiPhiPool":  {
                                     "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                     "User":  "aaronsace",
                                     "Worker":  "multipoolminer",
                                     "API_ID":  0,
                                     "API_Key":  ""
                                 },
                  "YiiMP":  {
                                "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                "User":  "aaronsace",
                                "Worker":  "multipoolminer",
                                "API_ID":  0,
                                "API_Key":  ""
                            },
                  "ZergPool":  {
                                   "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                   "User":  "aaronsace",
                                   "Worker":  "multipoolminer",
                                   "API_ID":  0,
                                   "API_Key":  ""
                               },
                  "Zpool":  {
                                "BTC":  "1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb",
                                "User":  "aaronsace",
                                "Worker":  "multipoolminer",
                                "API_ID":  0,
                                "API_Key":  ""
                            }
              },
    "Miners":  {

               }
}
```